src/renderer/views/play/customLobbies/customLobbies.vue

```diff
@@ -0,0 +1,205 @@
+<!--
```

Comment:
Better to move this back to the original directory. We get a better diff, but more importantly the ``lobby.vue`` file is currently dev-only since we implemented the Battle Drawer setup instead. Maybe later if we end up with a specific view for a lobby (i.e. once UI Team provides the spec) we can review if the structure needs to be separated or not like this.